src/API/Reports/Categories/DataStore.php

```diff
@@ -114,7 +114,16 @@ protected function get_sql_query_params( $query_args ) {
 			$sql_query_params['where_clause'] .= " AND ( {$order_status_filter} )";
 		}
 
-		$sql_query_params['where_clause'] .= " AND taxonomy = 'product_cat' ";
+		$where_clause = $sql_query_params['where_clause'] . " AND taxonomy = 'product_cat' ";
```

Comment:
What do you think about using the `$where_clause` variable instead of `$sql_query_params` before the `apply_filters()` call?

Reply:
Hmm.. on second thought - that would exclude the result of `get_time_period_sql_params()` from the filter. If that's desired perhaps we should switch up the variable, if not please disregard this comment. :)